glide.lock

```diff
@@ -12,7 +12,7 @@ imports:
   - uast
   - uast/ann
 - name: github.com/BurntSushi/toml
-  version: b26d9c308763d68093482582cea63d69be07a0f0
+  version: a368813c5e648fee92e5f6c30e3944ff9d5e8895
```

Comment:
Mmm, I'm not sure we should update all dependencies to their `~master` every time we update the SDK one. We probably should just update to the latest releases, trough that could depend on every dependency.